src/utils/validators/index.ts

```diff
@@ -43,7 +43,8 @@ const validatePercentage = (percent: string) => {
   return true;
 };
 
-const validateSlack = (v: string): boolean => v.match("(#|@).+") !== null;
+const validateSlack = (v: string): boolean =>
+  v.match("(#|@).+|(^S+$)") !== null;
```

Comment:
Not sure why we didn't already have them, but could you add some tests for this function in `validators.test.ts`? these tests are really easy to write and should only take like 2 minutes 🥺

Reply:
done

Reply:
I also narrowed down the regex to the more general one